lambda_code/src/models/errors.py

```diff
@@ -12,7 +12,7 @@ class Code(str, Enum):
     invalid = "invalid"
     server_error = "internal-server-error"
     invariant = "invariant"
-    invalid_resource = "invalid_resource"
+    unprocessable_entity = "unprocessable_entity"
```

Comment:
These are (supposed to be) FHIR OperationOutcome codes I think. 
http://hl7.org/fhir/R4B/codesystem-issue-type.html
Though some of the others seem to not exist in the FHIR docs either. I think that needs reviewing.

Reply:
Doing as https://nhsd-jira.digital.nhs.uk/browse/AMB-1953.